app/services/room_collection_service.py

```
@@ -243,6 +252,42 @@ def _extract_price(self, room: Dict) -> Optional[int]:
         # Use minPrice as the base price
         return min_max.get("minPrice")
 
+    def _calculate_capacity_range(
+        self,
+        parsed_range: Optional[List[int]],
```

Ruff (0.15.0)

[warning] Unused method argument: `parsed_range`

(ARG002)